public/app/themes/clarity/src/components/c-news-article/view.php

```diff
@@ -37,9 +40,12 @@
 
     <?php
         echo '<h1 class="o-title">Recent news</h1>';
-        foreach ($query->posts as $key => $post) {
+        $main_post = $post;
```

Comment:
Same question as for [c-article/view.php](https://github.com/ministryofjustice/intranet/pull/748/files#r1816478965)